app/org/sagebionetworks/bridge/services/email/ParticipantRosterProvider.java

```diff
@@ -81,6 +81,9 @@ String createParticipantTSV() {
         for (String attribute : study.getUserProfileAttributes()) {
             append(sb, StringUtils.capitalize(attribute), true);
         }
+        if (study.isHealthCodeExportEnabled()) {
+            append(sb, "Health Code", true);
```

Comment:
It is to be consistent, all of these header fields are strings and none are repeated anywhere other than this CSV method, so constants didn't seem necessary.